date: '2020-08-11'
sections:
  security_fixes:
    - '{% octicon "alert" aria-label="O ícone de alerta" %} **Crítico:** Foi identificada uma vulnerabilidade de execução de código remoto no GitHub Pages que poderia permitir que um invasor executasse comandos como parte da compilação de um site GitHub Pages. Este problema ocorria devido ao uso de uma dependência ultrapassada e vulnerável no processo de compilação das páginas. Para explorar esta vulnerabilidade, um invasor precisaria de permissão para criar e compilar um site GitHub Pages na instância do GitHub Enterprise Server.  Esta vulnerabilidade afetou todas as versões do GitHub Enterprise Server. Para mitigar esta vulnerabilidade, Kramdown foi atualizada para o endereço CVE-2020-14001. {% comment %} https://github.com/github/pages/pull/2835, https://github.com/github/pages/pull/2827 {% endcomment %}'
    - '**Alto:** Alto: Um invasor poderia injetar um argumento malicioso em um subcomando Git quando executado no GitHub Enterprise Server. Isso poderia permitir que um invasor substituísse arquivos arbitrários com conteúdo parcialmente controlado por usuário e potencialmente executar comandos arbitrários na instância do GitHub Enterprise Server. Para explorar essa vulnerabilidade, um invasor precisaria de permissão para acessar os repositórios dentro da instância GHES. No entanto, devido à outras proteções existentes, não conseguimos identificar um modo de explorar ativamente esta vulnerabilidade. Esta vulnerabilidade foi relatada pelo Programa de Recompensas por Bugs do GitHub Security. {% comment %} https://github.com/github/github/pull/150936, https://github.com/github/github/pull/150634 {% endcomment %}'
    - 'Os pacotes foram atualizados para as últimas versões de segurança. {% comment %} https://github.com/github/enterprise2/pull/21679, https://github.com/github/enterprise2/pull/21542, https://github.com/github/enterprise2/pull/21812, https://github.com/github/enterprise2/pull/21700 {% endcomment %}'
  bugs:
    - 'Um erro de configuração do Cônsul impediu que algumas tarefas de segundo plano fossem processadas em instâncias autônomas. {% comment %} https://github.com/github/enterprise2/pull/21463 {% endcomment %}'
    - 'O cálculo de alocação de memória de serviço poderia determinar uma alocação de memória incorreta ou ilimitada para um serviço resultando em desempenho de sistema pobre. {% comment %} https://github.com/github/enterprise2/pull/21689 {% endcomment %}'
    - 'A plataforma de virtualização para sistemas oVirt KVM não foi detectada corretamente, gerando problemas durante as atualizações. {% comment %} https://github.com/github/enterprise2/pull/21731, https://github.com/github/enterprise2/pull/21669 {% endcomment %}'
    - "A mensagem de erro para autenticação inválida com uma senha via linha de comando do Git não preencheu a URL vinculada à adição do token apropriado ou chave SSH. {% comment %} https://github.com/github/github/pull/149607, https://github.com/github/github/pull/149351 {% endcomment %}"
    - 'Criar um problema em um repositório de usuário que usa o recurso de modelo do problema pode falhar com um erro de servidor interno. {% comment %} https://github.com/github/github/pull/150173, https://github.com/github/github/pull/149445 {% endcomment %}'
    - 'Ocorreu uma falha ao acessar a seção *Explorar* com um erro de servidor interno de 500. {% comment %} https://github.com/github/github/pull/150512, https://github.com/github/github/pull/150504 {% endcomment %}'
    - 'Os problemas não puderam ser classificados por *Atualizações recentes* nos repositórios migrados para uma nova instância. {% comment %} https://github.com/github/github/pull/150688, https://github.com/github/github/pull/149330 {% endcomment %}'
    - 'O GitHub Connect estava usando um ponto obsoleto da API do GitHub.com. {% comment %} https://github.com/github/github/pull/150827, https://github.com/github/github/pull/150545 {% endcomment %}'
    - 'A coleta de métricas internas para trabalhos de segundo plano contribuiu para o uso desnecessário da CPU e da memória. {% comment %} https://github.com/github/github/pull/151182, https://github.com/github/github/pull/147695 {% endcomment %}'
    - 'A página 404 continha links de contato e links status do GitHub.com na nota de rodapé. {% comment %} https://github.com/github/github/pull/151315 {% endcomment %}'
    - 'Trabalhos de segundo plano para um recurso não publicado foram enfileirados e deixados sem processamento. {% comment %} https://github.com/github/github/pull/151395, https://github.com/github/github/pull/146248 {% endcomment %}'
  known_issues:
    - 'Em uma nova configuração do GitHub Enterprise Server sem qualquer usuário, um invasor pode criar o primeiro usuário administrador. {% comment %} https://github.com/github/enterprise2/issues/1889 {% endcomment %}'
    - 'As regras personalizadas do firewall não são mantidas em um upgrade. {% comment %} https://github.com/github/enterprise2/issues/2823 {% endcomment %}'
    - 'Arquivos LFS do Git [enviados através da interface Web](https://github.com/blog/2105-upload-files-to-your-repositories) são adicionados diretamente ao repositório e de maneira incorreta. {% comment %} https://github.com/github/github/issues/54684 {% endcomment %}'
    - 'Os problemas não podem ser fechados se contiverem um permalink para um blob no mesmo repositório em que o caminho do arquivo tem mais de 255 caracteres. {% comment %} https://github.com/github/github/issues/107731 {% endcomment %}'
    - 'Quando "Usuários podem pesquisar no GitHub.com" está habilitado com o GitHub Connect, os problemas nos repositórios privados e internos não estão incluídos nos resultados de pesquisa do GitHub.com. {% comment %} https://github.com/github/admin-experience/issues/571 {% endcomment %}'
    - 'Os alertas de segurança não são relatados ao fazer envio por push em um repositório na linha de comando. {% comment %} https://github.com/github/github/issues/143190 {% endcomment %}'
    - 'Os logs de auditoria podem ser atribuídos a 127.0.0.1 ao invés do endereço IP de origem real. (atualizado 2020-11-02) {% comment %} https://github.com/github/enterprise2/issues/21514 {% endcomment %}'
    - "Ocorre uma falha na configuração da permissão de um repositório para \"Triagem\" ou \"Manter\" com uma mensagem de erro."
    - 'Quando um nó de réplica está offline em uma configuração de alta disponibilidade, {% data variables.product.product_name %} ainda pode encaminhar solicitações de {% data variables.product.prodname_pages %} para o nó offline, reduzindo a disponibilidade de {% data variables.product.prodname_pages %} para os usuários.'
